compilers/gcc/BUILD

```diff
@@ -1,35 +1,43 @@
-
 # using -pipe causes spurious test-suite failures
-bad_flags -pipe &&
+CFLAGS=${CFLAGS/-pipe/} &&
+CXXFLAGS=${CXXFLAGS/-pipe/} &&
```

Comment:
`bad_flags` is standard in lunar and it removes *all* occurrences of the flag.
If `CFLAGS` contains the flag more than once, your code fails.

Please just rever this part.